indico/modules/users/templates/dashboard.html

```diff
@@ -82,6 +82,36 @@ <h3>{{ user.full_name }}</h3>
                         </a>
                     </div>
                 </div>
+                <div id="happeningCategories" class="dashboard-box">
```

Comment:
The `happeningCategories` ID is now duplicate, and still unused so let's just remove it in both places